src/Docker/Runner/DataLoader/WorkspaceProviderFactoryFactory.php

```diff
@@ -90,4 +96,14 @@ private function getWorkspaceFactoryForPersistentAbsWorkspace(Component $compone
             $connectionString
         );
     }
+
+    private function resolveWorkspaceBackendConfiguration(array $backendConfig = null)
+    {
+        if ($backendConfig === null) {
```

Comment:
vzhledem k tomu, že workspace backend akceptuje null, tak tohle je úplně zbytečná větev

Reply:
Pravdu mas.. nedoslo mi, ze ten `isset` si hezky poradi i s tim, kdyz je to `null` :+1: